packages/utilities/src/utilities/smoothScroll/smoothScroll.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Utility handles smoothScroll to the target element after OnClick
+ *
+ * @example
+ * import {smoothScroll} from '@carbon/ibmdotcom-utilities';
+ *
+ * smoothScroll(e);
```

Comment:
I think this is still a bit fuzzy. this is specifically for smooth scrolling to an anchor link on the page, so something like:

```
<a href="#anchorlinkname" onclick="smoothScroll(e)">lorem ipsum</a>
```